packages/dashboard/src/pages/Welcome/index.tsx

```diff
@@ -33,7 +32,7 @@ const IFrame = styled('iframe')(({ theme }) => ({
 export default function Welcome() {
     const iframeRef = useRef<HTMLIFrameElement | null>(null)
     const mode = useTheme().palette.mode
-    const navigate = useNavigate()
+    const [, setupSocialNetwork] = useSetupSocialNetwork()
```

Comment:
Why? Can't you just call `Services.SocialNetwork.setupSocialNetwork` directly? Why do you need a hooks?